build.gradle

```diff
@@ -15,12 +15,13 @@
  */
 
 plugins {
-    id 'org.ajoberstar.git-publish' version '2.0.0-rc.2' apply false
-    id 'org.curioswitch.gradle-curiostack-plugin'
+    id 'org.ajoberstar.git-publish' version '3.0.0-rc.1' apply false
+    id 'org.curioswitch.gradle-grpc-api-plugin'
+    id 'org.curioswitch.gradle-conda-plugin'
 }
 
 conda {
-    'miniconda2-build' {
+    'miniconda-build' {
```

Comment:
Phew looks like this all works with python 3 (good since 2 is permanently dead as of this year).